python-avd/pyavd/_anta/logs.py

```diff
@@ -95,6 +107,7 @@ class LogMessage(Enum):
     PATH_GROUP_NO_LOCAL_INTERFACES = "path group {path_group} skipped - No local interfaces found"
     PATH_GROUP_NO_STATIC_PEERS = "path group {path_group} skipped - No static peers configured"
     NO_STATIC_PEERS = "skipped - No static peers configured in any path groups"
+    IPv6_STATIC_PEER = "static peer {peer} skipped - ANTA does not support IPv6 static peer"
```

Comment:
Let's also add the path_group in the message for more context, similar to the other `PATH_GROUP_XXX` messages.
```suggestion
    PATH_GROUP_IPV6_STATIC_PEER = "static peer {peer} skipped - ANTA does not support IPv6 static peer"
```

Reply:
updated

Reply:
@MaheshGSLAB Please see my comment above the suggestion :) - Let's also add the path_group in the message for more context, similar to the other PATH_GROUP_XXX messages.

Reply:
sorry i missed the comment, updated now